admin/app/views/email/expiringSwitches.scala.html

```diff
@@ -1,25 +1,54 @@
-@(expiringImminently: Seq[conf.switches.Switch], expiringSoon: Seq[conf.switches.Switch])
+@import conf.switches.Switch
+@import _root_.jobs.ExpiringSwitches
 
-@showSwitch(switch: conf.switches.Switch) = {
-    <li>
-        <div>@{switch.name}</div>
-        <div>expires @{switch.sellByDate.get.toString("E dd MMM")} 23:59 UTC</div>
+@(switches: Seq[Switch])
+
+@showSwitch(switch: conf.switches.Switch, showExpirationDate: Boolean) = {
+    <li style="padding-bottom: 6px">
+        <div>
+            <strong>@{switch.name}</strong>
+            - @showOwners(switch.owners)
+            @if(showExpirationDate) { - <span style="color:grey;font-style: italic;">expires @{switch.sellByDate.get.toString("E dd MMM")} at 23:59 (London time)</span>}
+        </div>
         <div>@{switch.description}</div>
```

Comment:
But probably the info hierarchy could be tweaked so that the description has less visual weight than the expiry date.